src/sv_utils/src/sv_utils/benchmark_variant_filter.py

```diff
@@ -94,6 +94,7 @@ class Default:
     min_num_called_gt_hardy_weinberg = 5
     histogram_width = 0.9
     make_figures = _all_make_figures
+    sv_selector_size_ranges = get_truth_overlap.Default.sv_selector_size_ranges
```

Comment:
We do override these with CLI arguments.  I would rather leave this alone since it risks exposing bugs in the truth overlap code which we don't use and are likely going to deprecate.